command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
I am not sure what the compliance tests for CLI and runtime will do independently if they are separate. They are tied together.